crates/bevy_app/src/app.rs

```diff
@@ -42,12 +41,18 @@ bevy_utils::define_label!(AppLabel);
 /// }
 /// ```
 pub struct App {
+    /// Stores and exposes operations on [entities](bevy_ecs::entity::Entity), [components](bevy_ecs::component::Component), resources,
+    /// and their associated metadata.
```

Comment:
I think this should say why this field matters to `App`, not what a `World` does in general. That should be covered by the docs on the `World` type itself. This here doesn't seem like the place to explain what a `World` is.

Instead, this doc should be something more like:
> The main ECS `World` of the `App`.
>This stores and provides access to all the main data of the application. The systems of the `App` will run using this `World`. If additional separate `World`s and schedules are needed, you can use Sub-Apps.